.gitignore

```diff
@@ -1,2 +1,6 @@
 _site/
 /.quarto/
+.gitignore
```

Comment:
Need to remove this.

Reply:
Ops... My bad... What should I do? Remove the PR and push it again?

Reply:
Just need to accept suggestion. Sorry to not include before.
```suggestion
```